app/controllers/medicaid/insurance_current_controller.rb

```diff
@@ -2,5 +2,25 @@
 
 module Medicaid
   class InsuranceCurrentController < MedicaidStepsController
+    def update
+      @step = step_class.new(step_params)
+
+      if @step.valid?
+        if single_member_household?
```

Comment:
this whole method is a pattern we've implemented at least 3 times now, perhaps should start to think about extracting

Reply:
and now we did extract! woop woop

Reply:
✅